src/GramatykiGrafoweAGH/__init__.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
-from collections import defaultdict, Iterable
+from collections import defaultdict
+from collections.abc import Iterable
```

Comment:
Oops, my bad. We should import `Iterable` from `typing`. PyCharm sometimes gets it wrong.
```suggestion
from typing import Iterable
```